core/commands/bootstrap.go

```diff
@@ -21,21 +21,10 @@ var peerOptionDesc = "A peer to add to the bootstrap list (in the format '<multi
 
 var BootstrapCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
-		Tagline: "Show or edit the list of bootstrap peers.",
-		Synopsis: `
-ipfs bootstrap list             - Show peers in the bootstrap list
-ipfs bootstrap add <peer>...    - Add peers to the bootstrap list
-ipfs bootstrap rm <peer>... - Removes peers from the bootstrap list
-`,
-		ShortDescription: `
-Running 'ipfs bootstrap' with no arguments will run 'ipfs bootstrap list'.
-` + bootstrapSecurityWarning,
+		Tagline:          "Show or edit the list of bootstrap peers.",
+		ShortDescription: bootstrapSecurityWarning,
 	},
 
-	Run:        bootstrapListCmd.Run,
```

Comment:
If only we had aliases!

```
$ ipfs alias set bootstrap = bootstrap list
```

I can revert this though, if that's going to block this whole PR.